tests/baselines/reference/contextualTyping.errors.txt

```diff
@@ -226,16 +236,24 @@ tests/cases/compiler/contextualTyping.ts(230,5): error TS2322: Type '{}' is not
     }
     
     Point.origin = new Point(0, 0);
+          ~~~~~~
+!!! error TS2339: Property 'origin' does not exist on type '(x: any, y: any) => any'.

```

Comment:
This part of this test should probably be moved into a `.js` file, to preserve original intent.

Reply:
While I agree that moving this part into a .js file is the right thing, I don't understand why the ambient class should provide contextual types for the functions. At best they should merge, which is what they do now.

Reply:
Eh? Ok. I don't really have a strong opinion on it, since it feels like the kinda thing that maybe just feel out from other more intentioned choices before.

Reply:
Hm. Well, @DanielRosenwasser convinced me that contextual typing makes sense for the prototype method assignment, but not for the function declaration. I removed the conflicting function instead.